### DISCUSSION TITLE
Is RSS and AUC correlated with cell numbers? Recommendations for prioritizing and visualizing regulons?

### DISCUSSION THREAD

**Discussion by AmosFong1** [aertslab/scenicplus#419] [Q&A] [answered]
I ran SCENIC+ with 5 groups for comparison. I plot a custom dot plot heat map with TF-expression on a color scale, and gene based RSS on a size scale. I selected TF's that were differentially expressed using a pseudobulk-edgeR approach for visualization. I saw that the Gene based RSS was systematically lower in certain cell types for all TFs evaluated. Is RSS and AUC correlated with cell numbers?

On a similar note, can you please recommend metrics for visualizing in dotplot heat maps? Gene based AUC, Region based AUC, TF expression, gene based RSS, region based RSS? Do you have recommendation for how to prioritize regulons?

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @AmosFong1 

Yes the RSS is especially sensitive to cell numbers! I would recommend to scale the values across regulons in this case.
I often like to visualize Gene based and region based AUC or TF expression with either gene or region based AUC. 

Best,

Seppe

**Comment by AmosFong1** [comment]
Thank you for the quick and helpful answer!

**Reply by SeppeDeWinter** [reply]
You are welcome!

Best,

Seppe
